index.test-d.ts

```diff
@@ -12,4 +12,17 @@ expectType<elementReady.StoppablePromise<Element | undefined>>(promise);
 expectType<elementReady.StoppablePromise<HTMLDivElement | undefined>>(elementReady('div'));
 expectType<elementReady.StoppablePromise<SVGElement | undefined>>(elementReady('text'));
 
+
+const {stop} = elementReady.subscribe(".unicorn", e => null);
+elementReady.subscribe(".unicorn", e => null, {target: document});
```

Comment:
Use single-quotes

Reply:
Why doesn’t xo catch this?

Reply:
XO doesn't run on TS files unless you enable it explicitly. https://github.com/xojs/xo/issues/373 will enable us to automatically lint `.d.ts` files too.

Reply:
Indeed, I meant to say "let's just enable XO for TS files on this repo"

Reply:
Nah, I try to keep my boilerplate config as identical as possible between repos for ease of maintenance. I'd rather wait for XO.